Makefile

```diff
@@ -58,12 +56,6 @@ e2e: cmd/algorand-indexer/algorand-indexer
 e2e-filter-test: cmd/algorand-indexer/algorand-indexer
 	cd e2e_tests/docker/indexer-filtered/ && docker-compose build --build-arg GO_IMAGE=${GO_IMAGE} && docker-compose up --exit-code-from e2e-read
 
-e2e-filter-test-nightly: cmd/algorand-indexer/algorand-indexer
-	cd e2e_tests/docker/indexer-filtered/ && docker-compose build --build-arg GO_IMAGE=${GO_IMAGE} --build-arg CHANNEL=nightly && docker-compose up --exit-code-from e2e-read
-
-e2e-nightly: cmd/algorand-indexer/algorand-indexer
-	cd e2e_tests/docker/indexer/ && docker-compose build --build-arg GO_IMAGE=${GO_IMAGE} --build-arg CHANNEL=nightly && docker-compose up --exit-code-from e2e
```

Comment:
I guess these are removed because nightly is the new default? Does that mean the `make e2e` now runs the nightly channel instead of stable?

Reply:
yes, make e2e runs nightly by default. It gets overwritten if CHANNEL is set.

Reply:
Could you change the defaults so that it normally gives you the stable test and and the nightly job overrides the default?

Reply:
updated